indexer/httpclient.go

```diff
@@ -47,19 +47,19 @@ func (s *httpClient) Index(ctx context.Context, manifest *claircore.Manifest) (*
 		return nil, &clairerror.ErrBadManifest{err}
 	}
 
-	url := url.URL{
-		Scheme: s.addr.Scheme,
-		Host:   s.addr.Hostname(),
-		Path:   IndexAPIPath,
+	u, err := s.addr.Parse(IndexAPIPath)
```

Comment:
I don't see anything in GoDoc about the Parse method adding a Path field to the existing URL. This is how it works here?

Reply:
Yeah, it parses it in the context of the *net.URL object: [link](https://pkg.go.dev/net/url?tab=doc#URL.Parse)